### ISSUE TITLE
ScatterND Plugin missing for Torch to ONNX to TRT

### ISSUE THREAD

**Issue by KelvinCPChiu** [NVIDIA/TensorRT#736]
I am converting a Torch model to TRT. The original model could run but there are too many bounding box proposals. And I tried to add topK and argmax function to remove those low confidence bounding box. The conversion to ONNX is fine. But it failed when I am converting ONNX to TensorRT Engine. A scatterND plugin missing error message raised.

` 
__________________________________________________________________________
confidence := shape(batch, num_of_proposal)

_ , keep_idx = torch.topk(confidence, k=30, sorted=True, dim=1, largest=True)

oboxes = torch.zeros((batch, 30, 1, 4), dtype=torch.float32)
oconfs = torch.zeros((batch, 30, 80), dtype=torch.float32)

for b_idx in range(0, batch, 1):
        print(keep_idx.shape)
        oboxes[b_idx, :, :, :] = boxes[b_idx, keep_idx[b_idx], :, :]
        oconfs[b_idx, :, :] = confs[b_idx, keep_idx[b_idx], :]
__________________________________________________________________________
`

I think the scatterND plugin missing is resulting from the assignment of the for loop.
I would like to ask if there is anyway to bypass/solve this problem ?
Thanks.

**Comment by zhaosj98** [comment]
confidence := shape(batch, num_of_proposal)

_ , keep_idx = torch.topk(confidence, k=30, sorted=True, dim=1, largest=True)

obox_list = []
oconf_list = []
print(keep_idx.shape)
for b_idx in range(0, batch, 1):
  obox_list.append( boxes[b_idx, keep_idx[b_idx], :, :])
  oconf_list.append(confs[b_idx, keep_idx[b_idx], :])
oboxes = torch.stack(obox_list, dim=0)
oconfs = torch.stack(oconf_list, dim=0)

**Comment by KelvinCPChiu** [comment]
Thank you very much, @zhaosj98 ! It solved the scatterND problem. This makes me understand how ONNX to TRT works better.

**Comment by DonggeunYu** [comment]
@zhaosj98 Thank you 💯
